Automattic/jetpack
# Publicize: move it to plugins folder

~# Branched off from https://github.com/Automattic/jetpack/pull/20999~

As part of the https://github.com/Automattic/jetpack/issues/16816 task, this PR moves the publicize plugin to the proper place.

Related https://github.com/Automattic/jetpack/issues/16816

#### Changes proposed in this Pull Request:

* Move Publizice plugin to the proper code location

#### Jetpack product discussion


p1631196074241200-slack-

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Apply these changes
* Start to create a new post
* Check that the Publicize feature works as expected, compared with the current implementation.

<img src="https://user-images.githubusercontent.com/77539/132708763-30236d33-6a17-4d4c-8860-68887679f6d5.png" width="500" />
